docs/docs/validation-action.mdx

```diff
@@ -100,6 +107,51 @@ Then it will get a mapping of slots which were recently set and their values via
 Looping through this mapping of recently extracted slots, it will check if the slot is in `required_slots` then run the
 `validate_<slot name>` method if available for that slot.
 
-#### **Returns**
+##### **Returns**
 
   A list of `rasa_sdk.events.SlotSet` instances. See the [documentation for SlotSet events](sdk-events.mdx#slotset).
+
+
+## `FormValidationAction` class implementation
+
+`FormValidationAction` is a subclass of the `ValidationAction` Rasa SDK class and the abstract Python `ABC` class.
+`FormValidationAction` inherits most methods from `ValidationAction` class, however it overrides the `name` and
+`domain_slots` methods, implements a new method `next_requested_slot` and extends the implementation of `run` method.
+
+### Methods
+
+#### FormValidationAction.name
+
+The method `name` will raise a `NotImplementedError` exception if the bot custom action subclassing `FormValidationAction`
+does not return a custom name which follows this naming convention: `validate_<form name>`.
+
+#### FormValidationAction.required_slots
+
+The method `required_slots` will return the `domain_slots` which is a list of all slot names included in the form's
+`required_slots`. `domain_slots` is returned by the `domain_slots` method, which only takes
+`Domain` as an argument.
+
+#### FormValidationAction.next_requested_slot
+
+The method `next_requested_slot` will set the value of `REQUESTED_SLOT` to the next unset slot only if the
+`required_slots` method was overriden by the custom action subclassing `FormValidationAction`.
+
+If users didn't override `required_slots` then we'll let the `FormAction` within Rasa Open
+Source request the next slot, and the method will return `None`.
+
+The parameters the method requires are:
+- [dispatcher](sdk-dispatcher.mdx)
+- [tracker](sdk-tracker.mdx)
+- the bot's domain
+
+#### FormValidationAction.run
+
+The original implementation of the `ValidationAction.run` method is extended to add a call to the `next_requested_slot`
+method. The output of the `next_requested_slot` method call (if not `None`) is added to the list of events that `run`
+method returns.
+
+
+## How to subclass slot validation actions
+
+To learn more about how to subclass `ValidationAction` and `FormValidationAction` classes in your custom actions, please
+visit [Slot Validation Actions](https://rasa.com/docs/rasa/next/slot-validation-actions) in the Rasa Open Source docs.
```

Comment:
I think we had slightly different ideas of what the "sdk implementation" boundary was -  I was actually thinking almost everything currently under https://rasa.com/docs/rasa/next/slot-validation-actions goes here, because it is direct documentation on how to use rasa-sdk.

Then the  slot-validation-actions page on OSS would describe, action-server agnostically: 
* What is expected of the actions that are called automatically (`action_validate_<form_name>` and `action_validate_slot_mappings`)
* What is expected of custom actions referred to by slot mappings directly

So it would end up a fairly short page, almost all the details would move here. 

To be fair, all the actions docs pages have things in them that belong elsewhere. E.g. the default actions page correctly describes only the functionality and events returned by default actions, but then has a random sdk-specific overwriting example.

Reply:
Probably worth a separate issue for cleaning up the other action-server-specific docs by putting them in the  SDK docs

Reply:
> mostly delete the overlapping content and add a link instead

Regarding the link, that won't exist before the release I suspect - once this is merged and I update the rasa docs, how do I solve the link issue? Would the `main/unreleased` link exist and use that instead?

Reply:
mm I remember this being a problem before too. In which order will the release happen? OSS then SDK or other way around?

Reply:
It's the SDK first.

Reply:
Then it'll be fine, the link will exist when OSS is released.

Reply:
But will I be able to merge the PR with these changes to rasa `main`? usually `Test Docs` CI step fails if link doesn't exist.

Reply:
I guess though that's a problem for  getting the PR merged to release branch

Reply:
Ahaha ja same thought at same time.

Reply:
Is there an ignore flag for certain links?

Reply:
I don't know actually, maybe @m-vdb knows: we want to update Rasa docs with a link from SDK that won't exist before the release, is there a way to ignore this link during the CI checks?

Reply:
There is this: https://github.com/RasaHQ/rasa/blob/c2b5176f221c35be628cd0099ce486c4ce187108/docs/package.json#L113-L118
maybe best to point to main/unreleased though, that seems like a long term thing

Reply:
can you use the `/next/` URL here and clean it up post release instead of disabling the check?

Reply:
Yup, that sounds the best solution.